Windows/Debugger/CtrlDisAsmView.cpp

```diff
@@ -1352,14 +1355,176 @@ void CtrlDisAsmView::disassembleToFile() {
 	}
 }
 
+
+// The following member has been modified to accomodate "nicer-looking" output (according to my needs). Assumes maximum length of an instruction name is 9 or 10 chars.
+// What's the maximum length the params string could reasonably be?
 void CtrlDisAsmView::getOpcodeText(u32 address, char* dest, int bufsize)
 {
 	DisassemblyLineInfo line;
 	address = manager.getStartAddress(address);
-	manager.getLine(address,displaySymbols,line);
-	snprintf(dest, bufsize, "%s  %s",line.name.c_str(),line.params.c_str());
+	manager.getLine(address, displaySymbols, line);
+	
+	// Begin trace logger mods
+	std::string instructionName = std::string(line.name.c_str());
+	std::string parameters = std::string(line.params.c_str());
+
+	// Remove any extra \t characters at the beginning of parameters, because they exist for some reason when instructionName is long enough. 
+	if (parameters.find('\t') != parameters.npos) {
+		parameters.erase(0,1);
+	}
+
+	// Formatting: Maximum length of an instruction seems to be 9 characters. We want the instruction "column" to be uniformly 10 characters long.
+	while (instructionName.length() < 10) {
+		instructionName += " ";
+	}
+	instructionName += "\t"; // Plus an extra tab character for increased readability
+
+	//DEBUG_LOG(CPU, parameters.c_str());
+
+	// Setting the paremeters "column" to have 32 characters uniformly.
+	while (parameters.length() < 32) {
+		parameters += " ";
+	}
+	// removed the space between %s and %s since i pre-formatted them. 
+	snprintf(dest, bufsize, "%s%s", instructionName.c_str(), parameters.c_str());
```

Comment:
`snprintf` and similar is made to format string, I suggest to search on google if there is syntax to format your string as you want with it directly first (there is a lot in it, too much to be worth learning rather than searching when you need tho').

In this case: https://stackoverflow.com/questions/293438/left-pad-printf-with-spaces